src/scripts/templates.ts

```diff
@@ -321,7 +321,7 @@ const templates = {
     const opt = new Option(label, value, false, active);
 
     if (customProperties) {
```

Comment:
FYI, but this isn't a sufficient test for if a javascript object is empty

Reply:
Thank you for the feedback. I'll fix the tests and update the PR.

Reply:
@Xon I've updated the tests. Could you please let me know if this is correct. I'm not really great with JS as I majorly work on backends. Thank you.